.goreleaser.yaml

```diff
@@ -7,17 +7,31 @@ before:
     # you may remove this if you don't need go generate
     - go generate ./...
 builds:
-  - env:
-      - CGO_ENABLED=0
+  - binary: saturn-L2-node
+    id: saturn-L2-node
     goos:
       - linux
       - windows
-      - darwin
     ignore:
     - goos: windows
       goarch: arm64
     main: ./cmd/saturn-l2
-    binary: saturn-L2-node
+    env:
+      - CGO_ENABLED=0
+  - binary: saturn-L2-node
+    id: saturn-L2-node-macos
```

Comment:
create a separate binary with its own `id`, which can be targeted in the `signs` option